### DISCUSSION TITLE
permission denied after upgrade

### DISCUSSION THREAD

**Discussion by c094728** [git-for-windows/git#3614] [Q&A] [answered]
I upgraded from git version 2.28.0.windows.1 to 2.34.1
After the upgrade I get error pulling from any of our repos (private server)
Permission denied (keyboard-interactive,publickey).
fatal: Could not read from remote repository.

I reinstalled original version and then it works again.
We use ssh key to access the server.
A clone command is like this
git clone ssh://username%40company.com@alm.company.com:2222/repo/project.git
any ideas?

**Comment by PhilipOakley** [comment, marked as answer]
You mention ssh. There has been a recent (v2.33.1) change to Openssh which removed some encryption methods from the default list of acceptable methods. See https://github.com/git-for-windows/git/issues/3468 for extra details. Hopefully it helps.

**Reply by sunilchadha1973** [reply]
Thanks for pointing out, I had to downgrade to 2.32.0 to get around ssh error. I tried other work around but it did not work.
